cloudigrade/account/migrations/0001_initial.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.0.2 on 2018-02-12 18:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Account',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
For a migration you need to specify the primary key autofield, but not for the model itself? Basically, I need to learn about migrations.

Reply:
The migration is a generated file, if you do not specify a primary key in your model, django will provide one for you.